server/src/main/java/lambda/MessageHandler.java

```diff
@@ -52,8 +54,50 @@ public CreateMessageResponse createMessage(CreateMessageRequest request, Context
     }
 
     public GetMessagesResponse getMessages(GetMessagesRequest request, Context context) {
+        AmazonDynamoDBClient client = new AmazonDynamoDBClient().withEndpoint("https://dynamodb.us-west-2.amazonaws.com");
```

Comment:
handler don't trigger db. handler accepts the request and transfer jobs to models